### ISSUE TITLE
ib cable inventory does not flag cables with serial_number=N/A

### ISSUE THREAD

**Issue by thanh-lam** [IBM/CAST#682]
**Describe the bug**
The ib_cable inventory had a cable with serial_number=N/A but the command `standalone_ib_and_switch_collection` didn't display a WARNING. Also, this cable record should be in the inventory file `/var/log/ibm/csm/inv/bad_ib_cable_records.txt`.

**To Reproduce**
1. Run `/opt/ibm/csm/sbin/standalone_ib_and_switch_collection`.
2. Watch the displayed messages: (No WARNING about bad ib_cable records)
``` 
[root@f4n05 sbin]# ./standalone_ib_and_switch_collection

UFM reported 189 IB records.
This report from UFM can be found in 'ufm_ib_cable_output_file.json' located at '/var/log/ibm/csm/inv'

UFM reported 9 switch records.
This report from UFM can be found in 'ufm_switch_output_file.json' located at '/var/log/ibm/csm/inv'
WARNING: 2 Switches found with 'N/A' serial numbers and have been removed from CSM inventory collection data.
These records copied into 'bad_switch_records.txt' located at '/var/log/ibm/csm/inv'
...
```
3. Change to `/var/log/ibm/csm/archive/` and search for "N/A" in the ib_cable records:
```
# grep "N/A" csm_ib_cable_history.archive.2019-05-14.json
{ "type":"db-csm_ib_cable_history", "data":{"history_time": "2019-05-14 14:29:58.471150", "comment": "comment", "port_s1": "37", "name": "ec0d9a03001576b0_37:ec0d9a03001576b8_1", "archive_history_time": null, "guid_s1": "ec0d9a03001576b0", "guid_s2": "ec0d9a03001576b8", "discovery_time": "2019-04-30 09:37:38.585673", "part_number": "N/A", "severity": "Info", "width": "IB_4x", "length": "N/A", "type": "NA - UNKNOWN", "port_s2": "1", "serial_number": "N/A", "operation": "U", "identifier": "Unknown", "revision": "N/A", "collection_time": "2019-05-14 14:29:58.471150"} }
```

**Expected behavior**
The command `standalone_ib_and_switch_collection` should display a WARNING about the bad ib_cable. And, the file `/var/log/ibm/csm/inv/bad_ib_cable_records.txt` should contain the bad record.

**Screenshots**

**Environment (please complete the following information):**
 - Machine [IST alt. cluster f4n05]
 - Version [CSM 1.6.0]

**Additional context**
The command `csm_ib_cable_query` also incorrectly picked up "N/A" as a legitimate serial_number.
```
# ./csm_ib_cable_query -s "N/A"
---
Total_Records: 1
RECORD_1:
  serial_number:  N/A
  discovery_time:  2019-05-14 15:06:02.040102
  collection_time: 2019-05-14 15:06:02.040102
  comment:         comment
  guid_s1:         248a070300c6eea0
  guid_s2:         248a070300c6eea8
  identifier:      Unknown
  length:          N/A
  name:            248a070300c6eea0_37:248a070300c6eea8_1
  part_number:     N/A
  port_s1:         37
  port_s2:         1
  revision:        N/A
  severity:        Info
  type:            NA - UNKNOWN
  width:           IB_4x
...
```

**Issue Source:**
ib cable inventory tests in IST env.

**Comment by NickyDaB** [comment]
for later notes:
```
---
# ib inventory successful
...
# total ib inventory collected: 113
# new ib records inserted into database: 101
# old ib records updated in database: 12
# old ib records removed from the database: 0
---

```

**Comment by NickyDaB** [comment]
I believe that UFM restAPIs changed the output of a missing serial number in the most recent update. Previously we searched for `NA` now it displays as `N/A`. We now look for both forms in our code. Well, in my feature branch where I am working on this issue.

**Comment by NickyDaB** [comment]
This has been merged into master

**Comment by thanh-lam** [comment]
Verified the fix in CSM build 1.6.0-2520. It now recognizes cables with serial_number = N/A and cables with missing info (serial_number = NOT AVAILABLE).

**Comment by NickyDaB** [comment]
@thanh-lam . Awesome. Thanks.
